build/config/compiler/BUILD.gn

```diff
@@ -263,6 +263,7 @@ config("strict_warnings") {
       "-Wheader-hygiene",
       "-Wshorten-64-to-32",
       "-Wformat-type-confusion",
+      "-Wthread-safety",
```

Comment:
how does it behave w/ GCC ?

Reply:
That's only a Clang feature. It won't work with GCC. That's why it's in the `if (is_clang)` block.